contentScript.js

```diff
@@ -1,84 +1,7 @@
 var parsed_model = JSON.parse(atob(model))
 var parser = new DOMParser();
 
-const buttonCheckboxMapping = {
-  age18checkbox: {
-    labelId: "age18checkboxlabel",
-    label: "Age 18+",
-    checked: false
-  },
-  age45checkbox: {
-    labelId: "age45checkboxlabel",
-    label: "Age 45+",
-    checked: false
-  },
-  covishieldcheckbox: {
-    labelId: "covishieldcheckboxlabel",
-    label: "Covishield",
-    checked: false
-  },
-  covaxincheckbox: {
-    labelId: "covaxincheckboxlabel",
-    label: "Covaxin",
-    checked: false
-  },
-  sputnikcheckbox: {
-    labelId: "sputnikcheckboxlabel",
-    label: "Sputnik V",
-    checked: false
-  },
-  freecheckbox: {
-    labelId: "freecheckboxlabel",
-    label: "Free",
-    checked: false
-  },
-  paidcheckbox: {
-    labelId: "paidcheckboxlabel",
-    label: "Paid",
-    checked: false
-  }
-}
 
-let mobilenumber = window.localStorage.getItem("mobile");
-let state_name = window.localStorage.getItem("state");
-let district_name = window.localStorage.getItem("district");
-let first_5_pin_digits = window.localStorage.getItem("pincode");
-let allow_multiple = window.localStorage.getItem("allow_multiple") === "true" ? true : false;
-let searchByDistrictFlag = window.localStorage.getItem("searchpref") === "district" ? true : false;
-let keeptryingcontinuously = window.localStorage.getItem("keeptryingcontinuously") === "true" ? true : false;
-let timeslotind = window.localStorage.getItem("timeslot");
-let enableAutoRefresh = window.localStorage.getItem("autorefresh") === "true" ? true : false;
-let enableautoconfirm = window.localStorage.getItem("autoconfirm") === "true" ? true : false;
-let minavailability = window.localStorage.getItem("minavailability");
-let selected_button_checkbox = window.localStorage.getItem("selectedbuttoncheckboxes")
-let center_prefs_string = window.localStorage.getItem("centerprefs");
-let center_prefs_dirty = center_prefs_string ? center_prefs_string.split(",") : "";
-let autorefreshinterval = window.localStorage.getItem("autorefreshinterval");
-let skipdays = window.localStorage.getItem("skipdays");
-
-const sleep = (delay) => {
-  return new Promise((resolve) => {
-    setTimeout(() => {
-      resolve(true);
-    }, delay);
-  })
-}
-
-if (isNaN(parseInt(skipdays))) {
-  skipdays = 0;
-} else {
-  skipdays = parseInt(skipdays);
-}
-let center_prefs = [];
-for (let i = 0; i < center_prefs_dirty.length; i++) {
-  let t = center_prefs_dirty[i].trim();
-  if (t !== "") {
-    center_prefs.push(t);
-  }
-}
-if (center_prefs.length === 0) {
-  center_prefs = [""];
-}
 
 let checked_buttons = [];
```

Comment:
This line down to line 23 also need to be added to the form.js file. Because of this the checkboxes are not checked
![image](https://user-images.githubusercontent.com/30574783/120107617-6128c000-c17f-11eb-90e0-2838373bf9bc.png)

Reply:
Okay.. will do